Form/EventListener/ResizePolyFormListener.php

```diff
@@ -84,7 +85,12 @@ public function __construct(array $prototypes, array $options = array(), $allowA
                 $defaultType = $type;
             }
 
-            $typeKey = $type instanceof FormTypeInterface ? $type->getName() : $type;
+            $typeKey = $type;
+
+            if ($type instanceof FormTypeInterface) {
```

Comment:
I'm not sure, that this is the right place to deprecate FormType instances. `getInnerType()` always returns a `FormTypeInterface` object.